lib/person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :name,
+              :interests,
+              :supplies
+
+  def initialize(person_info)
+    @name      = person_info[:name]
+    @interests = person_info[:interests]
+    @supplies  = {}
+  end
+
+  def add_supply(supply, amount)
+    sum = 0
+    sum += amount
+    @supplies[supply] = amount
+    if @supplies.keys.include?(supply)
+      @supplies[supply] = sum
+    end
+  end
```

Comment:
On line 15 you are overwriting whatever is in `@supplies[supply]` before you check if it already exits and add to the current amount. 

```suggestion
  def add_supply(supply, amount)
    if @supplies.keys.include?(supply)
      @supplies[supply] += amount
    else
      @supplies[supply] = amount
    end
  end
```